backend/src/apiserver/server/util_test.go

```diff
@@ -345,6 +345,32 @@ func TestGetUserIdentity(t *testing.T) {
 	assert.Equal(t, "user@google.com", userIdentity)
 }
 
+func TestGetUserIdentityError(t *testing.T) {
+	md := metadata.New(map[string]string{"no-identity-header": "user"})
+	ctx := metadata.NewIncomingContext(context.Background(), md)
+	_, err := getUserIdentity(ctx)
+	assert.NotNil(t, err)
```

Comment:
nit: can we also assert the error message here to make sure the error came from missing header

Reply:
Done! Thanks